pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -118,7 +118,7 @@ This page contains the configuration parameters for both DC/OS Enterprise and DC
 
 | Parameter                    | Description                                                                                                                                                       |
 |------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [enable_mesos_input_plugin](#enable-mesos-input-plugin)    | Indicates whether to enable Telegraf's Mesos input plugin to collect Mesos metrics from Mesos masters and agents. Default value is `false`. |
+| [enable_mesos_input_plugin](#enable-mesos-input-plugin)    | Indicates whether to enable Telegraf's Mesos input plugin to collect Mesos metrics from Mesos masters and agents. Default value is `true`. |
```

Comment:
Add a note to say mesos metrics plugin is not enabled by default in 1.12, but is in 1.13.

Copy-paste the below
<p class="message--note"><strong>NOTE: </strong>DC/OS 1.13 supports *mesos metrics* plugin by default</p>**